### PR TITLE
k6 0.24.0 (New Formula)

### LINKED ISSUE #380: Create Homebrew-core formula
See: https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md

Grab the mac zip from here:
https://github.com/loadimpact/k6/releases/tag/v0.18.0

Also, update the readme since it will no longer be necessary to tap:
https://github.com/loadimpact/k6/blob/master/README.md

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Closes https://github.com/loadimpact/homebrew-k6/issues/6
Closes https://github.com/loadimpact/k6/issues/380

Question from upstream repo:

> If we remove this formula after it's in homebrew-core, would any existing k6 users that have installed it automagically get the new k6 versions when we release them? Or would they have to re-install k6 from the new formula?

### CODE REVIEW

**Comment by chenrui333** [comment]
Question from upstream repo:

> If we remove this formula after it's in homebrew-core, would any existing k6 users that have installed it automagically get the new k6 versions when we release them? Or would they have to re-install k6 from the new formula?

**Comment by SMillerDev** [comment]
If you add a rename (json file in the root) it'll migrate all users that update. (Meaning you could remove the old formula)

**Comment by chenrui333** [comment]
Sounds pretty good!, I will add that one now. cc @@na--

**Comment by chenrui333** [comment]
@SMillerDev Updated the json file.

**Comment by chenrui333** [comment]
@fxcoudert I have filed a new PR in the to-migrate homebrew repo:
`https://github.com/loadimpact/homebrew-k6/pull/7`

**Comment by chenrui333** [comment]
@fxcoudert @SMillerDev This can be merged and @na-- will do the merge right after the merge in here. :)

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request closed without merging**
